### ISSUE TITLE
Add Guild Wars 2 script

### ISSUE THREAD

**Issue by PayBas** [PayBas/GameTooltips#7]
http://www.gw2db.com/tooltips

**Comment by AkumanuG** [comment]
Sadly, gw2db is not up to date. The traits they list haven't been around since last june, and the expansion content is missing entirely.

**Comment by Nixitro** [comment]
I agree with AkumanuG. gw2db.com is outdated, they dont update approx since march'14. GameTooltips with GW2 will be an awesome update.

**Comment by SeluneArkady** [comment]
Are there any news about tooltips for gw2 script ?

**Comment by miri4ever** [comment]
Looks not still wairing and no guess who will continue developing this nice thing.

**Comment by Galixte** [comment]
Please, this kind of allusion shouldn't be here.

Paybas is gone to other occupations, Now, Sajaki does what he can.

We have to be patiente, if you aren’t, learn PHP/HTML/CSS languages to help him. ;)

Moreover, gw2db.com is responsible to maintain the GW2 script, if you want it ask them.
